driver/normalizer/annotation.go

```diff
@@ -171,14 +168,20 @@ var AnnotationRules = On(jdt.CompilationUnit).Roles(uast.File).Descendants(
 
 	// Operators
 	On(jdt.InfixExpression).Roles(uast.Expression, uast.Binary, uast.Operator).Self(
-		On(HasProperty("operator", "+")).Roles(uast.Add),
-		On(HasProperty("operator", "-")).Roles(uast.Substract),
-		On(HasProperty("operator", "*")).Roles(uast.Multiply),
-		On(HasProperty("operator", "/")).Roles(uast.Divide),
-		On(HasProperty("operator", "%")).Roles(uast.Modulo),
+		On(HasProperty("operator", "+")).Roles(uast.Add, uast.Arithmetic),
+		On(HasProperty("operator", "-")).Roles(uast.Substract, uast.Arithmetic),
+		On(HasProperty("operator", "*")).Roles(uast.Multiply, uast.Arithmetic),
+		On(HasProperty("operator", "/")).Roles(uast.Divide, uast.Arithmetic),
+		On(HasProperty("operator", "%")).Roles(uast.Modulo, uast.Arithmetic),
```

Comment:
I'm not sure if you've to scape the % here.

Reply:
Not needed.